src/Converter.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Objects;
+
+/**
+ * Created by ken-linux2 on 24/07/16.
+ */
+public class Converter {
+
+    Account account=new Account();
+    Product product=new Product();
+    Channel channel=new Channel();
+    AdGroup adGroup=new AdGroup();
+
+    public Channel returnLineAsChannel(String oneLine[]){
+//        System.out.print(oneLine);
+    setAccountData(oneLine[6],oneLine[12],oneLine[1]) ;
+    setProductData(null,oneLine[5],oneLine[13],oneLine[14],oneLine[15],oneLine[16]);
+    setAdGroupData(oneLine[8],oneLine[3],oneLine[9],oneLine[11]);
+    setChannelData(oneLine[0],oneLine[2],oneLine[4]);
+
+    return channel;
+}
+
+    public void setAccountData(String name,String vendorClass,String id){
```

Comment:
You can name this buildAccount. You can also create a constructor that accept these fields. We usually don't want more than 3-4 args in the constructor as it becomes bug-prone (we you use a setter it is easier to see that is being set)